config/spaceruinblacklist.txt

```diff
@@ -66,13 +68,3 @@
 #_maps/RandomRuins/SpaceRuins/waystation.dmm
 #_maps/RandomRuins/SpaceRuins/whiteshipdock.dmm
 #_maps/RandomRuins/SpaceRuins/whiteshipruin_box.dmm
-#_maps/RandomRuins/SpaceRuins/forgottenship.dmm
-#_maps/RandomRuins/SpaceRuins/hellfactory.dmm
-#_maps/RandomRuins/SpaceRuins/space_billboard.dmm
-#_maps/RandomRuins/SpaceRuins/spinwardsmoothies.dmm
-#_maps/RandomRuins/SpaceRuins/dangerous_research.dmm
-#_maps/RandomRuins/SpaceRuins/the_faceoff.dmm
-#_maps/RandomRuins/SpaceRuins/atmosasteroidruin.dmm
-#_maps/RandomRuins/SpaceRuins/prey_pod.dmm
-#_maps/RandomRuins/SpaceRuins/travelers_rest.dmm
-#_maps/RandomRuins/SpaceRuins/whiteshipruin_box.dmm
```

Comment:
Something's gone wrong again here, these should not be deleted.

Reply:
The duplicate ones? I only removed stuff that appeared twice and shifted the rest in alphabetically. Unless that's intended in which case FUCK